scripts/deploy.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+TARGET_DIR='gh-pages'
+TARGET_BRANCH='master'
+TARGET_REPO_URL="https://$GITHUB_AUTH_TOKEN@github.com/chartjs/chartjs.github.io.git.git"
```

Comment:
is this line supposed to end in `.git.git`? I would have guessed you'd have `.git` only once